Sources/SourceControl/GitRepository.swift

```diff
@@ -16,13 +16,13 @@ import func POSIX.getenv
 import enum POSIX.Error
 
 enum GitRepositoryProviderError: Swift.Error {
-    case gitCloneFailure(url: String, path: AbsolutePath)
+    case gitCloneFailure(url: String, path: AbsolutePath, errorOutput: String)
```

Comment:
No no, I meant we catch and handle this error in `Commands/Error.swift` so it is printed better for the user. For example this can be caught like this in the `handle(error:)` method in that file.
```swift
    ...
    case GitRepositoryProviderError. gitCloneFailure(let url, _, let output):
        print(error: "Failed to clone \(url.asString):\n" + output)
    ...
```